src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+
+    The garbage collector doesn't track write-ahead logs that are to be
+    relayed to anonymous replicas, such as:
+
+    - Anonymous replicas added as a part of a cluster configuration
+    (see `replication.anon`).
+    -CDC (Change Data Capture) that retrieves data using anonymous replication.
+
+    In case of a replica or CDC downtime, the required write-ahead logs
+    can be removed. As a result, such a replica needs to be rebootstrapped.
+    You can use wal.retention_period to prevent such issues.
+
+    Note that `wal.cleanup_delay` option also sets the delay used to prevent
+    the Tarantool garbage collector from removing write-ahead logs. The
+    difference is that the garbage collector doesn't take into account
+    `wal.cleanup_delay` if all the nodes in the replica set are up and running,
+    which may lead to the removal of the required write-ahead logs.
+]])
+
+-- }}} wal configuration
+
+-- }}} Instance descriptions
+
+-- {{{ Cluster descriptions
+
+local L = {}
+
+L[''] = 'Cluster configuration'
+
+-- {{{ conditional configuration
+
+L['conditional'] = format_text([[
+    The `conditional` section defines the configuration parts that apply to
+    instances that meet certain conditions.
+]])
+
+L['conditional.*'] = format_text([[
+    `conditional.if`: Defines a section where configuration options apply only
+    if specified conditions are met.
+
+    Conditions can include `tarantool_version` and support comparison operators
+    (`>`, `<`, `>=`, `<=`, `==`, `!=`), as well as logical operators
+    (`||`, `&&`) and parentheses for grouping.
+]])
+
+L['conditional.*.*'] = format_text([[
+    `conditional.if.*`: Defines a specific condition and the configuration
+    options applied when the condition is met.
+
+    Each condition compares the `tarantool_version` using operators
+    (`<`, `>`, `<=`, `>=`, `==`, `!=`) and supports defining labels or other
+    options that are applied only if the condition holds true.
+]])
+
+-- }}} conditional configuration
+
+-- {{{ groups configuration
+
+L['groups'] = format_text([[
+    This section provides the ability to define the full topology
+    of a Tarantool cluster.
+]])
+
+L['groups.*'] = format_text([[
+    A group of replicasets.
+
+    The following rules are applied to group names:
+
+    - The maximum number of symbols is 63.
+    - Should start with a letter.
+    - Can contain lowercase letters (a-z).
+    - Can contain digits (0-9).
+    - Can contain the following characters: -, _.
+]])
+
+L['groups.*.replicasets'] = 'Replica sets that belong to this group.'
+
+L['groups.*.replicasets.*'] = format_text([[
+    A group of instances.
+
+    Note that the rules applied to a replica set name are the same as for
+    groups. Learn more in `groups.<group_name>`.
+]])
+
+L['groups.*.replicasets.*.instances'] = format_text([[
+    Instances that belong to this replica set.
+]])
+
+L['groups.*.replicasets.*.instances.*'] = 'An instance name.'
```

Comment:
It describes a key, but we should describe values of a mapping, IMHO. I would say

> An instance definition.

Reply:
Done.